src/index.html

```diff
@@ -1,13 +1,61 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
     <title>Product cards</title>
-    <link rel="stylesheet" href="./styles/index.scss">
+
+    <link rel="preconnect"
+          href="https://fonts.googleapis.com"
+    >
+
+    <link rel="preconnect"
+          href="https://fonts.gstatic.com"
+    >
+
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+          rel="stylesheet"
+    >
+
+    <link rel="stylesheet"
+          href="./styles/index.scss"
+    >
+
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card"
+         data-qa="card"
+    >
+      <img src="images/imac.jpeg"
+           alt="product item imac"
+           class="card__photo"
+      >
+      <h2 class="card__title">APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)</h2>
+      <span class="card__description">Product code: 195434</span>
+      <div class="card__review">
+
+        <ul class="card__review-stars stars stars--4">
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+        </ul>
+
+        <span class="card__review-raiting">Reviews: 5</span>
+      </div>
+
+      <div class="card__price">
+        <span class="card__price-value">Price:</span>
+        <span class="card__price-amount">$2,199</span>
+      </div>
+
+      <a class="card__buttom"
+         data-qa="hover">Buy
+      </a>
+
+    </div>
```

Comment:
```suggestion
      <a 
        class="card__buttom"
        data-qa="hover"
      >
        Buy
      </a>
    </div>
```

Reply:
Sorry. I never used to write with lint before, but on the contrary, I used to write without any rules, that is why I have these mistakes, I apologize once again